.github/workflows/deploy_api.yml

```diff
@@ -1,12 +1,8 @@
 name: Build & Publish API snapshot to data.covidactnow.org
 
-#######################################################################################
-###
-### YOU DON'T NEED TO MODIFY THIS FILE TO KICK OFF A SNAPSHOT FROM YOUR BRANCH ANYMORE.
-###
-### See: https://github.com/covid-projections/covid-data-model/#api-snapshots
-###
-#######################################################################################
+# Use a concurrency group to make sure we don't try to have multiple workflows
+# run with the hosted runner at the same time.
+concurrency: gce-runner
```

Comment:
Not 100% sure if we strictly need this.  Since there's only one gce-runner, everything runs serially on the runner, but since now we have some self-hosted jobs within this workflow I think there could be weird race conditions.  Like one VM could be trying to start the runner VM while another one is trying to stop the runner VM or something.  By using a concurrency group, I'm hoping that is avoided.

Reply:
Yeah, good call 🙏 can't hurt to be safe!